README.md

```diff
@@ -50,6 +50,42 @@ have dependencies on other algebras which must be implemented.
     - Two promises are equivalent when they yield equivalent values.
     - Two functions are equivalent if they yield equivalent outputs for equivalent inputs.
 
+## Type signature notation
+
+The type signature notation used in this document is described
+[here](https://sanctuary.js.org/#types). In summary:
+
+1. _New types can be created via type constructors._
+    - Type constructors take one or more type arguments. For example, the type
+    "array of strings" is written as `Array String`, and the type constructor
+    `Array` can be defined as `Array :: Type -> Type`.
```

Comment:
@davidchambers I've re-written the section based on this info and some research: https://github.com/justin-calleja/fantasy-land/tree/issue-222#type-signature-notation